recipes/awsbatch_config.rb

```diff
@@ -57,5 +57,7 @@
 else
   python_package "aws-parallelcluster" do
     version node['cfncluster']['cfncluster-version']
+    retries 3
```

Comment:
are we sure this is supported for python_package?

Reply:
It [inherits](https://github.com/poise/poise-python/blob/master/lib/poise_python/resources/python_package.rb#L102) from chef package, which has the retry.

Btw I also tested it manually with this snippet
```
python_package "aws-parallelcluster" do
    retrie 3
    retry_delay 5
end
```
and it worked